bodhi/client/__init__.py

```diff
@@ -35,28 +69,13 @@ def updates():
 
 
 @updates.command()
+@add_options(new_edit_options)
 @click.argument('builds')
-@click.option('--user', envvar='USERNAME')
-@click.option('--password', prompt=True, hide_input=True)
-@click.option('--type', default='bugfix', help='Update type', required=True,
-              type=click.Choice(['security', 'bugfix',
-                                 'enhancement', 'newpackage']))
-@click.option('--notes', help='Update description')
 @click.option('--notes-file', help='Update description from a file')
```

Comment:
Couldn't you want to edit the type of an update ? Same for notes ? 
I don't know I don't have much user experience of bodhi, maybe it's better to give the user the choice.